lib/api.js

```diff
@@ -32,6 +32,10 @@ api.declare({
   scopes:      [['secrets:set:<name>']],
   title:       'Set Secret',
   stability:    'stable',
+  cleanPayload: payload => {
```

Comment:
Oh, that's silly.  Your method is clearer, but I think there's a method involving "..." that's even better.. let me play.